src/config-management.ts

```diff
@@ -136,11 +136,16 @@ export function _isUnavailablePeriodValid(period: Period): boolean {
 		period
 		&& period.startAt
 		&& period.endAt
-		&& period.startAt.length === period.endAt.length
-		&& _isDateStringValid(period.startAt)
-		&& _isDateStringValid(period.endAt)
+		/* Both have year, or both have not */
+		&& (period.startAt.year == null) === (period.endAt.year == null)
+		&& _isPeriodMomentValid(period.startAt)
+		&& _isPeriodMomentValid(period.endAt)
 		/* If no year specified, the order can be reversed: the next year will be take for endAt */
```

Comment:
/* If the year value isn't specified, endAt can precede startAt, and doing so will set the endAt year value to the following year */